components/automate-ui/src/app/page-components/client-runs-search-bar/client-runs-search-bar.component.ts

```diff
@@ -70,7 +70,8 @@ export class ClientRunsSearchBarComponent implements OnChanges {
 
   ngOnChanges(changes: SimpleChanges) {
     if (changes.filterValues) {
-      this.suggestions = List<string>(changes.filterValues.currentValue);
+      const listValues = this.handleFiltersValues(changes.filterValues.currentValue);
+      this.suggestions = List<string>(listValues);
```

Comment:
This is a pretty small change in terms of code necessary for the fix but this PR introduces a new variable and function. Could this change be simplified to:

```
this.suggestions = List<string>(compact(changes.filterValues.currentValue));
```

Reply:
sure I will make these changes.

Reply:
I added the changes.